Component/Badge/badge.css

```diff
@@ -0,0 +1,75 @@
+@import url("../../Utility CSS/navbar.css");
+
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: space-around;
+  align-items: center;
+  padding: 2rem;
+}
+
+figure {
+  position: relative;
+  margin: 8px;
+}
+.avatar-xl {
+  height: 17rem;
+}
+
+.round-img {
+  border-radius: 50%;
+}
+
+.avatar-lg {
+  height: 10rem;
+}
+.avatar-md {
+  height: 6rem;
+}
+
+.avatar-sm {
+  height: 3rem;

```

Comment:
Run prettier on this in order to indent the code properly;

Reply:
sure